projects/dendrites/profiling/forward_profile.py

```diff
@@ -24,68 +24,81 @@
 import torch.autograd.profiler as profiler
 
 from models import DendriticMLP, SparseMLP
+from nupic.research.frameworks.pytorch.model_utils import count_nonzero_params
 from nupic.research.frameworks.pytorch.models.common_models import StandardMLP
 
 
-def func(model, device, dendrite=False):
+def func(model, device, input_size, dendrite=False):
     use_cuda = device.type == "cuda"
+    dummy_tensor = torch.rand((1024, input_size), device=device)
```

Comment:
Before, we had the input dim to non-dendrite network be 21 since the input dim would be (dim(x) + dim(context)), and in that case, dim(x) = 11 and dim(context) == 10. Maybe this is more fair, since the input going into the network is the same (just incorporated in different manners)?